graphprot/NeuralNet.py

```diff
@@ -22,16 +22,24 @@ class NeuralNet(object):
     def __init__(self, database, Net,
                  node_feature=['type', 'polarity', 'bsa'],
                  edge_feature=['dist'], target='irmsd',
-                 batch_size=32, percent=[0.8, 0.2], index=None):
+                 batch_size=32, percent=[0.8, 0.2], index=None, database_eval = None):
 
         # dataset
         dataset = HDF5DataSet(root='./', database=database, index=index,
                               node_feature=node_feature, edge_feature=edge_feature,
                               target=target)
+        PreCluster(dataset, method='mcl')
 
         train_dataset, valid_dataset = DivideDataSet(
             dataset, percent=percent)
 
+        if database_eval :
```

Comment:
could you change the the condition to if `database_eval is not None` ? That would help readability of the code.